.github/workflows/ci.yml

```diff
@@ -87,6 +87,7 @@ jobs:
       project-name: './build/InstrumentationLibraries.proj'
       project-build-commands: '-p:RunningDotNetPack=true -p:ExposeExperimentalFeatures=false'
       code-cov-name: 'Instrumentation-Stable'
+      tfm-list: '[ "net6.0", "net7.0", "net8.0" ]'
```

Comment:
Curious, why is this needed here? Can this info be deduced from existing configuration? Ideally, TFM info should be in a central place so adding future TFM and retiring legacy TFM would only touch one single file.

Reply:
It is temporary change. Steps ` build-test-instrumentation-stable:` and `build-test-instrumentation-experimental` will be fully removed from the settings when Instrumentation.AspNetCore package is also migrated to the contrib. I would not care about this change so much.

If we speaking more generally, tfm's  are defined in two places right now:
1. Common.props - for .NET definitions what to build an how to pack production projects
2. Build definitions (mostly Component.BuildTets.yml) to define frameworks to execute tests in the pipeline.

Adding one more layer on top of it, to unify this two things could be artificial and potentially require introducing custom parsing. As long as we have _only 2 places_ it should be fine to keep it as is.

Expected changes happen once per year (together with .NET releases).

Reply:
Got it, thanks!